scanners/nuclei/parser/parser.js

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2021 iteratec GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+async function parse(fileContent) {
+  // Only 0 when the target wasn't reachable
+  if (fileContent.length === 0) {
+    return [];
+  }
+
+  const jsonResult = readJsonLines(fileContent);
+
+  return jsonResult.map((finding) => {
+    return {
+      name: finding.info.name,
+      description:
+        "The name of the nuclei rule which triggered the finding: " +
+        finding.templateID,
+      location: finding.host,
+      severity: getAdjustedSeverity(finding.info.severity.toUpperCase()),
+      category: finding.templateID,
+      attributes: {
+        metadata: finding.meta || null,
+        ip: finding.ip || null,
+        timestamp: finding.timestamp || null,
+        matcher_name: finding.matcher_name || null,
+        matched: finding.matched || null,
+        extracted_results: finding.extracted_results || null,
+        type: finding.type || null,
+        tags: finding.tags || null,
+        reference: finding.reference || null,
+        author: finding.author || null,
+      },
+    };
+  });
+}
+
+function getAdjustedSeverity(severity) {
+  return severity === "CRITICAL"
+    ? "HIGH"
+    : severity === "INFO"
+    ? "INFORMATIONAL"
+    : severity === "UNKNOWN"
+    ? "INFORMATIONAL"
+    : severity;
+}
```

Comment:
Adjust function to make codeclimate happy
```suggestion
function getAdjustedSeverity(severity) {
    if(severity === "CRITICAL") 
         return "HIGH";
    if(severity === "INFO" || severity === "UNKNOWN")
         return "INFORMATIONAL";
    return severity;     
}
```